src/locale/locales/ro/messages.po

```diff
@@ -212,122 +217,122 @@ msgstr "{count} elemente necitite"
+msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} alți} few {{formattedAuthorsCount} alți} other {{formattedAuthorsCount} alți}}</0> v-au urmărit"
 
 #: src/view/com/notifications/NotificationFeedItem.tsx:327
 msgid "{firstAuthorLink} and <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}}</0> liked your custom feed"
-msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {un alt utilizator} few {alți {formattedAuthorsCount} utilizatori} other {alți {formattedAuthorsCount} de utilizatori}}</0> au apreciat feed-ul tău personalizat"
+msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} alți} few {{formattedAuthorsCount} alți} other {{formattedAuthorsCount} alți}}</0> v-au apreciat fluxul personalizat"
 
 #: src/view/com/notifications/NotificationFeedItem.tsx:223
 msgid "{firstAuthorLink} and <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}}</0> liked your post"
-msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {un alt utilizator} few {alți {formattedAuthorsCount} utilizatori} other {alți {formattedAuthorsCount} de utilizatori}}</0> au apreciat postarea ta"
+msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} alți} few {{formattedAuthorsCount} alți} other {{formattedAuthorsCount} alți}}</0> v-au apreciat postarea"
 
 #: src/view/com/notifications/NotificationFeedItem.tsx:247
 msgid "{firstAuthorLink} and <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}}</0> reposted your post"
-msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {un alt utilizator} few {alți {formattedAuthorsCount} utilizatori} other {alți {formattedAuthorsCount} de utilizatori}}</0> au redistribuit postarea ta"
+msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} alți} few {{formattedAuthorsCount} alți} other {{formattedAuthorsCount} alți}}</0> v-au repostat postarea"
 
 #: src/view/com/notifications/NotificationFeedItem.tsx:351
 msgid "{firstAuthorLink} and <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} other} other {{formattedAuthorsCount} others}}</0> signed up with your starter pack"
-msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {un alt utilizator} few {alți {formattedAuthorsCount} utilizatori} other {alți {formattedAuthorsCount} de utilizatori}}</0> s-au înscris folosind pachetul tău de început"
+msgstr "{firstAuthorLink} și <0>{additionalAuthorsCount, plural, one {{formattedAuthorsCount} alți} few {{formattedAuthorsCount} alți} other {{formattedAuthorsCount} alți}}</0> s-au înregistrat cu pachetul dvs. de pornire"
```

Comment:
We are using the informal language ("tău", not "dumneavoastră" or "dvs."). See https://github.com/bluesky-social/social-app/pull/6456#issuecomment-2498593586